src/stylesheets/components/_concept.scss

```diff
@@ -177,12 +177,30 @@
     text-align: center;
     width: 14%;
 
+    &::before {
+      border: 14px solid transparent;
+      border-bottom: 0;
+      border-left: 0;
+      border-radius: 0 0 15px 15px;
+      border-right-color: $white;
+      bottom: -2px;
+      content: '';
+      height: 0;
+      right: 14px;
+      margin-right: -27px;
+      margin-top: -13.5px;
```

Comment:
With sub-pixel rendering, some browsers will round up while others round down. See if you can do either `13` or `14` here.